datafusion/tests/sql/joins.rs

```diff
@@ -419,32 +419,32 @@ async fn cross_join_unbalanced() {
 
     // the order of the values is not determinisitic, so we need to sort to check the values
     let sql =
-        "SELECT t1_id, t1_name, t2_name FROM t1 CROSS JOIN t2 ORDER BY t1_id, t1_name";
+        "SELECT t1_id, t1_name, t2_name FROM t1 CROSS JOIN t2 ORDER BY t1_id, t1_name, t2_name";
```

Comment:
Previously, since I was using SPMS and heap sort to do in_mem_sort, I find the sort non-stable.  for the current situation that I change back to combine then sort method, the behavior is unchanged. 

However, I think the test here against stableness is out of the scope for SQL sort operator?